scripts/emergency/toggle_pool_features.js

```
@@ -0,0 +1,262 @@
+                    if (pool.pool_info.pool_identifier) { 
+                        allPoolIdentifiers.push(pool.pool_info.pool_identifier);
+                    } else {
+                        console.warn("Found pool data entry without a 'pool_identifier' field:", pool);
+                    }
+                }
+                console.log(`Fetched ${pools.length} pools in this page. Total fetched so far: ${allPoolIdentifiers.length}`);
+
+                if (pools.length === limit) {
+                    // Get the pool_identifier of the last fetched pool for pagination
+                    const lastPool = pools[pools.length - 1];
+                    if (lastPool && lastPool.pool_identifier) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)